recipes/nss/all/conanfile.py

```diff
@@ -0,0 +1,204 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os, glob
+
+
+class NSSConan(ConanFile):
+    name = "nss"
+    license = "MPL-2.0"
+    homepage = "https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Network Security Services"
+    topics = ("network", "security", "crypto", "ssl")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
```

Comment:
```suggestion
    default_options = {"shared": True, "fPIC": True}
```

I suggest using "shared" True by default, as the static is invalid. Otherwise, any package which depends on nss, will if not explicit asking for static

Reply:
That would make the hook fail, and it's not necessary with this line right ?

https://github.com/conan-io/conan-center-index/pull/7864/files#diff-ece292fe38b97350cf102f6556e9fe24fdb29615b54888310e65818858fcf6d7R34